### PR TITLE
K8SPG-25 Add PMM

### PR DESCRIPTION
**Pull request by nonemax**

### CODE REVIEW

File: internal/operator/cluster/clusterlogic.go [hunk 1]
```diff
@@ -303,7 +303,10 @@ func getClusterDeploymentFields(clientset kubernetes.Interface,
 	for _, v := range tablespaceVolumes {
 		supplementalGroups = append(supplementalGroups, v.SupplementalGroups...)
 	}
-
+	sc := operator.GetPodSecurityContext(supplementalGroups)
+	if cl.Spec.PMM.Enabled {
```
Comment (hors) [inline internal/operator/cluster/clusterlogic.go:307]: please don't ignore settings from  supplementalGroups option

**Review by hors — CHANGES_REQUESTED** [review]

**Review by qJkee — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | failed |
| scaling | failed |

commit: https://github.com/percona/percona-postgresql-operator/pull/30/commits/b3d8807a69df547f2e696faff83e3aa516d460b6
image: `perconalab/percona-postgresql-operator:PR-30-b3d8807a-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-30-b3d8807a-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-30-b3d8807a-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-30-b3d8807a-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-30-b3d8807a-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-30-b3d8807a-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| scaling | passed |

commit: https://github.com/percona/percona-postgresql-operator/pull/30/commits/dc8c00748bbf22c4a230122fc5272166f1dc53ed
image: `perconalab/percona-postgresql-operator:PR-30-dc8c0074-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-30-dc8c0074-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-30-dc8c0074-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-30-dc8c0074-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-30-dc8c0074-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-30-dc8c0074-pgo-deployer`

**Review by hors — APPROVED** [review]

**Review by cap1984 — APPROVED** [review]

File: deploy/pmm-secret.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+data:
+  password: YWRtaW4=
+  username: YWRtaW4=
+kind: Secret
+metadata:
+  name: hippo-pmm-secret
```
[the remark this answers is not in the document]
Comment (hors) [reply deploy/pmm-secret.yaml:7]: @nonemax please fix it in main branch too

[the remark this answers is not in the document]
Comment (nonemax) [reply deploy/pmm-secret.yaml]: done https://github.com/percona/percona-postgresql-operator/pull/30/commits/3eeb1afa10016e0ef4cd40e98d6e4f71684b7e12

**Pull request merged**